include/boost/geometry/algorithms/crosses.hpp

```diff
@@ -30,7 +30,8 @@
 
 #include <boost/geometry/geometries/concepts/check.hpp>
 
-#include <boost/geometry/algorithms/detail/relate/relate.hpp>
+#include <boost/geometry/algorithms/relate.hpp>
+#include <boost/geometry/algorithms/detail/relate/relate_impl.hpp>
```

Comment:
It's sometimes used, yes. I have nothing against changing that name, previously it was `relate_base`. In this file there is a base class for a specialization/dispatched algorithm which could be implemented by falling back to relate. So it could e.g. be `algorithm_relate` or `relate_dispatch_base` but I don't have an idea which I think is the best. If you have an idea for a better name I'll change it.